packages/select/test/multiSelectTests.tsx

```diff
@@ -109,6 +109,60 @@ describe("<MultiSelect>", () => {
         assert.isTrue(handleRemove.calledOnceWithExactly(TOP_100_FILMS[3], 1));
     });
 
+    // NOTE: If customTarget is not supplied, MultiSelect will use requestAnimationFrame
+    // for the PopoverInteraction causing a delay that breaks tests. To get around this,
+    // after the click event is simulated you can await for a brief moment then call wrapper.update() to
+    // get latest state, otherwise the wrapper will have stale state and props.
```

Comment:
Slightly worded it better, but going to keep this comment since we're nesting it within requestAnimationFrame

> I'd recommend handling the special case early and returning and leaving the existing code as is. right now the onInteraction call is moved outside of the requestAnimationFrame and I don't want to worry about what changes that could cause